acceptance/lib/acceptance/bolt_command_helper.rb

```diff
@@ -18,7 +18,16 @@ def bolt_command_on(host, command, flags = {}, opts = {})
 
       case host['platform']
       when /windows/
-        execute_powershell_script_on(host, bolt_command, opts)
+        # Set the PATH environment variable before running the Bolt command. Ideally this
```

Comment:
I think you can do `host.add_env_var('PATH', new_path)` which actually it looks like we're one of the only ones that already does this in our setup 😅 [in Bolt acceptance](https://github.com/puppetlabs/bolt/blob/b77d7e7f2f458824fcbccad78367a0f113638463/acceptance/setup/common/pre-suite/010_install_ruby.rb), [Github search](https://github.com/search?q=org%3Apuppetlabs+add_env_var+path+path%3Aacceptance%2Fsetup&type=Code)

Reply:
@lucywyman Ya, that doesn't work. I'm not sure if it's some sort of config for the VM, but the path isn't preserved between PS sessions. Each beaker method that connects to the host runs in a new PS session, so the path won't have whatever changes were made with `host.add_env_var` when we run the Bolt command.